lib/quadstore.ts

```diff
@@ -24,7 +24,7 @@ import {
   TSVoidResult,
 } from './types/index.js';
 import assert from 'assert';
-import events from 'events';
+import {EventEmitter} from 'events';
```

Comment:
My project does not have `esModuleInterop` set. For some reason, just this import was causing an error.

Reply:
That's weird but I have heard of stuff like this before. I haven't encountered this myself but happy to merge this change if it helps.